pandas/tests/extension/test_categorical.py

```diff
@@ -216,9 +174,21 @@ def _compare_other(self, s, data, op, other):
         else:
             return super()._compare_other(s, data, op, other)
 
-
-class TestParsing(base.BaseParsingTests):
-    pass
+    @pytest.mark.xfail(reason="Categorical overrides __repr__")
+    @pytest.mark.parametrize("size", ["big", "small"])
+    def test_array_repr(self, data, size):
+        super().test_array_repr(data, size)
+
+    @pytest.mark.xfail(
+        reason="Looks like the test (incorrectly) implicitly assumes int/bool dtype"
+    )
+    def test_invert(self, data):
+        super().test_invert(data)
+
+    @pytest.mark.xfail(reason="TBD")
```

Comment:
Any better reason that could be specified here?

Reply:
i havent figured out whats going on here, was hoping to do that in a future pass